.github/workflows/test.yml

```
@@ -0,0 +1,36 @@
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Run tests
+        run: pnpm test
+
+      - name: Build project
+        run: pnpm build
+
+      - name: Test CLI (dry run)
+        run: node dist/cli.js --provider dummy --days 7
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)